netlify.toml

```diff
@@ -15,7 +15,7 @@ HUGO_VERSION = "v0.74.2"
 HUGO_ENV = "production"
 
 [context.deploy-preview]
-command = "hugo --gc --minify --buildFuture -b $DEPLOY_PRIME_URL"
+command = "hugo --gc --minify --buildFuture -b \"${DEPLOY_PRIME_URL}/\""
```

Comment:
Added "/" slash
https://github.com/womenengjp/womeneng.jp/issues/3#issuecomment-767563122

`$DEPLOY_PRIME_URL` is the url for the preview site.
ref: https://docs.netlify.com/configure-builds/environment-variables/#deploy-urls-and-metadata